src/main/java/com/google/sps/data/SampleData.java

```diff
@@ -195,6 +196,7 @@ public void addToStudentScheduledSessionsByEmail(String email, TutorSession tuto
     /** Deletes the given TutorSession from the scheduled sessions array of the given student. */
     public void deleteFromStudentScheduledSessionsByEmail(String email, TutorSession tutoringSession) {
         Student student = getStudentByEmail(email);
+        System.out.println(email);
```

Comment:
Do we still need these print statements here? Was this to check that the email exists?

Reply:
Yeah this is not necessary anymore. I'll remove it